docs/faq.md

```diff
@@ -0,0 +1,19 @@
+# Frequently Asked Questions 
+
+#### Why did we use Alevin-fry for processing? 
+
+In an effort to allow users to easily compare data available on the portal to their own processed data, we aimed to process all of the data using a method that is comparable to widely used pipelines.
+In our own benchmarking, we have found that Alevin-fry offers a faster, more memory efficient approach to processing single-cell and single-nuclei RNA-sequencing data than the commonly utilized 10X Genomics' [Cell Ranger,](https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/using/count) without sacrificing accuracy. 
+Alevin-fry uses approximately 12-16 GB of memory per sample and completes alignment and quantification in less than an hour, while Cell Ranger uses up to 25-30 GB of memory per sample taking anywhere from 2-8 hours to align and quantify one sample.
```

Comment:
I think it is important here to mention what mode we are using. Partly because if we used sketch mode we could get these numbers much lower, and I don't want to make people question the benchmarking difference they may see with other descriptions of alevin-fry.
```suggestion
In the configuration that we are using ("selective alignment" mapping to a human transcriptome that includes introns), Alevin-fry uses approximately 12-16 GB of memory per sample and completes mapping and quantification in less than an hour. 
By contrast, Cell Ranger uses up to 25-30 GB of memory per sample and takes anywhere from 2-8 hours to align and quantify one sample.
```